Binance.Net/Clients/UsdFuturesApi/BinanceRestClientUsdFuturesApiTrading.cs

```diff
@@ -294,8 +294,9 @@ public async Task<WebCallResult<BinanceUsdFuturesOrder>> EditOrderAsync(string s
                 { "symbol", symbol },
                 { "side", EnumConverter.GetString(side) },
                 { "quantity", quantity.ToString(CultureInfo.InvariantCulture) },
-                { "price", price.ToString(CultureInfo.InvariantCulture) },
             };
+            parameters.AddOptionalParameter("price", price?.ToString(CultureInfo.InvariantCulture));
```

Comment:
According to the documentation the price is required for USD futures

Reply:
I noticed that, it's a bit strange because the parameter itself is there and mentions:

> only avaliable for LIMIT/STOP/TAKE_PROFIT order; can be set to OPPONENT/ OPPONENT_5/ OPPONENT_10/ OPPONENT_20: /QUEUE/ QUEUE_5/ QUEUE_10/ QUEUE_20; Can't be passed together with price

I'll do some testing to see whether it gets accepted.

Reply:
I have tested it using the locally generated nuget package and the call does get accepted.